File: src/Python/easy/string/problem3019.py
class Solution:
    def countKeyChanges(self, s: str) -> int:
        s = s.lower()
        counter = 0
        for i in range(len(s)):
            if i + 1 < len(s):
                if s[i] != s[i + 1]:
                    counter += 1
        return counter 

s = "aAbBcC"

solution = Solution()
solution.countKeyChanges(s)

File: src/JavaScript/easy/string/problem3019.js
/**
 * @param {string} s
 * @return {number}
 */
var countKeyChanges = function (s) {
  s = s.toLowerCase();
  let counter = 0;
  for (let i = 0; i < s.length; i++) {
    if (i + 1 < s.length) {
      if (s[i] != s[i + 1]) {
        counter++;
      }
    }
  }
  return counter
};

// let s = "aAbBcC"
let s = "AaAaAaaA";

countKeyChanges(s);
